parsons/bill_com/bill_com.py

```diff
@@ -0,0 +1,316 @@
+            dev_key and sessionId added.
+        """
+        return {"devKey": self.dev_key,
+                "sessionId": self.session_id,
+                "data": json.dumps(data)}
+
+    def post_request(self, data, action, object_name):
+        """
+        `Args:`
+            data: dict
+                A dictionary containing the payload to be sent in the request.
+                The dev_key and sessionId should not be included as they are
+                dealt with separately.
+            action: str
+                The action to be taken on the given object.
+                Possible values are "List", "Read", "Create", and "Send".
+            object_name: str
+                The id of the object. Possible values are "User", "Customer", and "Invoice".
+
+        `Returns:`
+            A dictionary containing the JSON response from the post request.
+        """
+
+        if action == 'Read':
+            url = "%sCrud/%s/%s.json" % (self.api_url, action, object_name)
+        elif action == 'Create':
+            data['obj']['entity'] = object_name
+            url = "%sCrud/%s/%s.json" % (self.api_url, action, object_name)
+        elif action == "Send":
+            url = "%s%s%s.json" % (self.api_url, action, object_name)
+        else:
+            url = "%s%s/%s.json" % (self.api_url, action, object_name)
+        payload = self.get_payload(data)
+        response = requests.post(url=url, data=payload, headers=self.headers)
+        return response.json()
+
+    def get_request_response(self, data, action, object_name, field="response_data"):
+        """
+        `Args:`
+            data: dict
+                A dictionary containing the payload to be sent in the request.
+                The dev_key and sessionId should not be included as they are
+                dealt with separately.
+            action: str
+                The action to be taken on the given object.
+                Possible values are "List", "Read", "Create", and "Send".
+            object_name: str
+                The id of the object. Possible values are "User", "Customer", and "Invoice".
+            field: str
+                The JSON field where the response data is stored. Defaults to "response_data".
+
+        `Returns:`
+            A dictionary containing the choosen field from the JSON response from the post request.
+        """
+        return self.post_request(data, action, object_name)[field]
+
+    def get_user_list(self, start_user=0, max_user=999, **kwargs):
+        """
+        `Args:`
+            start_user: int
+                The index of first user to return. Starts from 0 (not 1).
+            max_user: str
+                The index of the max user to return
+            **kwargs:
+                Any other fields to pass
+
+        `Returns:`
+            A list of dictionaries of user information for every user from start_user to max_user.
```

Comment:
The Parsons convention is to wrap the response in a `Table` object when returning multiple records.

Reply:
This also applies to the other `list_` methods below.